### DISCUSSION TITLE
Difference from Zathura and Okular?

### DISCUSSION THREAD

**Discussion by AmeerArsala** [ahrm/sioyek#1224] [Q&A]
What's the difference between Sioyek, a riced-up Zathura (i.e. with the optional dependencies as described [here](https://wiki.archlinux.org/title/Zathura) and straight up raw Okular?

**Comment by panangam** [comment]
I think Okular is more akin to Adobe Acrobat: a generalist PDF viewer that an average can do annotations, printing, reading, etc using a mouse. Sioyek has a much more minimalist UI, with basically no button at all, but you can do most things using keyboard. It also has a smaller but tighter set of features geared specifically towards reading scientific papers and textbooks built into the app (reference following, Google Scholar searching, portal, visual mark, etc), whereas with Okular you will probably need to use a lot of plugins to do something similar. Though one advantage of Okular I've found is the annotation system follow the PDF standards, so it is more interoperable with other PDF viewers/annotators. Sioyek uses its own kind of database.  

Having tried Zathura for about 5 minutes, I'd say one big difference is the existence of command palette in Sioyek, so I can search for commands by names. With Zathura it seems I have to memorize all the keybindings which is not for me. I find the UI to be very similar though.
